crates/fuel-core/src/service/adapters/import_result_provider.rs

```diff
@@ -37,38 +37,42 @@ impl ImportResultProvider {
 impl ImportResultProvider {
     pub fn result_at_height(
```

Comment:
The mentioned issue was only related to this part of the code: https://github.com/FuelLabs/fuel-core/pull/2004#discussion_r1663230067

All other changes are not needed. All other places uses `None` because it means no prefix or not starting block